.github/README-pt-BR.md

```diff
@@ -90,7 +90,7 @@ Código e documentação copyright 2018 [B.C.Rikko](https://github.com/BcRikko).
 
 ## Desenvolvimento
 
-Se você quiser nos ajudar com o projeto, nós agradecemos contribuições de todos os tipos! Verifique nosso [`CONTRIBUTING-pt-BR.md`][contributing-document] para mais detalhes em como você pode ajudar a tornar o NES.css incrível!
+Se você quiser nos ajudar com o projeto, nós agradecemos as contribuições de todos os tipos! Verifique nosso [`CONTRIBUTING-pt-BR.md`][contributing-document] para mais detalhes como você pode ajudar a tornar o NES.css incrível!
```

Comment:
Hey, dude, what's up?

I think when you say "para mais detalhes como você pode ajudar..." it is a confusing phrase, but if you say "para mais detalhes de como você pode ajudar...", it sounds better.

Reply:
Hello. Yes, I agree.
But this PR is kind of dead for a year now, so if you want I think you can make a new one correcting all this mistakes (and other ones if you find those).
Thanks 👍